src/utils/github.ts

```diff
@@ -41,10 +42,80 @@ function getGithubGQLClient(): typeof graphql {
   });
 }
 
-export async function getRepositoryPaths(repository: string, dir = 'docs'): Promise<string[]> {
+export const GithubRESTClient = new Octokit({
+  auth: `token ${getGitHubToken()}`,
+  baseUrl: 'https://api.github.com',
+});
```

Comment:
this will only use the first token since it's a static const and only initializes once - this is why I made `GitHubGQLClient` a function -> `getGithubGQLClient`